fhir-bulkimportexport-webapp/src/main/java/META-INF/batch-jobs/FhirBulkExportChunkJob.xml

```diff
@@ -15,6 +15,7 @@
                     <property name="fhir.search.pagesize" value="#{jobParameters['fhir.search.pagesize']}"/>
                     <property name="fhir.typeFilters" value="#{jobParameters['fhir.typeFilters']}"/>
                     <property name="fhir.exportFormat" value="#{jobParameters['fhir.exportFormat']}"/>
+                    <property name="incomingUrl" value="#{jobParameters['incomingUrl']}"/>
```

Comment:
is this related to issue #300 at all, or its just a "bonus" ?

Reply:
unfortunately, it's related - it's interpreting the incomingUrl inorder to form the target URL in a reference lookup

Reply:
ah, ok, makes sense.  glad you thought through all the ramifications of the changes...i might have missed this impact to export